internal/tiltfile/tiltfile.go

```diff
@@ -28,23 +28,26 @@ func init() {
 }
 
 type TiltfileLoader interface {
-	Load(ctx context.Context, filename string, matching map[string]bool, logs io.Writer) (manifests []model.Manifest, global model.Manifest, configFiles []string, warnings []string, err error)
+	Load(ctx context.Context, filename string, matching map[string]bool, logs io.Writer) (manifests []model.Manifest, global model.Manifest, configFiles []string, warnings []string, builtinCallCounts map[string]int, err error)
```

Comment:
this api seems increasingly unmanageable.

can we dependency-inject TiltfileLoader, and make it call analytics directly?

Reply:
I was thinking it'd be nice to not need to worry about analytics just to parse a file. I guess we kind of have to either way, either in the ctor or the return value.

How about making Load return a `ProcessedTiltfile` and pulling all the return values into that? It's currently pretty ugly even without `builtinCallCounts`

Reply:
i would be fine with ProcessedTiltfile! 

But the builtinCallCounts would still seem like a weird thing to return from this function, since it's still very coupled to the analytics interface.